src/Silverback.Integration.Kafka/Messaging/Messages/KafkaStatisticsEvent.cs

```diff
@@ -13,14 +13,11 @@ namespace Silverback.Messaging.Messages
     /// </remarks>
     public class KafkaStatisticsEvent : IMessage
     {
-        public KafkaStatisticsEvent(string json)
-        {
-            Json = json;
-        }
+        public KafkaStatisticsEvent(string statistics) => Raw = statistics;
 
         /// <summary>
-        ///     Gets the statistics JSON (see https://github.com/edenhill/librdkafka/blob/master/STATISTICS.md)
+        ///     Gets the statistics in raw JSON, as they are provided from the underlining librdkafka (see https://github.com/edenhill/librdkafka/blob/master/STATISTICS.md).
         /// </summary>
-        public string Json { get; }
+        public string Raw { get; }
```

Comment:
I don't like to have the name of the format as property. Raw seems to be a better name wdyt?

Reply:
Btw. I think it would be great to expose the deserialized statistics. Wdyt? I will create the deserialization mechanism anyway and we could ship it with silverback.

Started it already: https://github.com/msallin/KafkaStatistics/blob/master/WorkerService1/Program.cs#L81

Reply:
I like `Raw` better, thanks.

About the deserialized statistics I find it a good idea if:
* the format is kept fairly backward compatible
  * also, does it depend on the server or is it given by librdkafka?
  * how do we (automatically) test it and ensure that it keeps working with newer versions of kafka / librdkafka?
* we make the deserializer fault tolerant (obviously) and thus we still provide the raw version

Having better, more csharpy, attribute names would be a plus.

Question: why don't we open an issue at Confluent to provide the deserialized version in the library already? Just to check with them whether that's a good idea and why they aren't doing that already.